src/components/FilterOptions.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import * as Types from "../utils/Types";
+import {
+  handleInput,
+  handleVisibilityToggle,
+  handleSearchBarToggle,
+} from "../utils/EventHandlers";
+
+const FilterOptions = (props: Types.FilterOptionsProps) => {
+  const [state] = props.stateManager;
+  return (
+    <div className="filter-options">
+      <input
+        className="searchbar"
+        placeholder="Enter institution, lab, or address"
+        onChange={handleInput(props.stateManager)}
+      />
+      <button onClick={handleVisibilityToggle(props.stateManager)}>
```

Comment:
Make this a check box with the text "Filter table by visibility"

Reply:
When the searchbar contains a query and the visibility checkbox is checked, should I take the intersection of searchbar query and the visiibility filters?